test/src/endpoints/account-endpoints.spec.js

```diff
@@ -66,6 +67,34 @@ describe('getAccountV1', () => {
   })
 })
 
+describe('updateCustomerAccountV1', () => {
+  test("update the customer account's details", () => {
```

Comment:
Personally I would just write 'update the customer account details'. I think it flows better and then we can move to using single quotes. Bit of a nitpicky one not essential so i'll leave it up to you 👍